ui/src/testing/factories/nodes.ts

```diff
@@ -356,6 +365,64 @@ export const podNumaNode = define<PodNumaNode>({
   vms: () => [],
 });
 
+export const podResource = define<PodResource>({
+  allocated_other: 2,
+  allocated_tracked: 1,
+  free: 3,
+});
+
+export const podMemoryResource = define<PodMemoryResource>({
+  general: podResource,
+  hugepages: podResource,
+});
+
+export const podNetworkInterface = extend<Model, PodNetworkInterface>(model, {
+  name: "eth0",
+  numa_index: 0,
+  virtual_functions: podResource(),
+});
+
+export const podVM = extend<Model, PodVM>(model, {
+  hugepages_backed: false,
+  memory: 4068,
+  pinned_cores: [0, 2],
+  system_id: "abc123",
+  unpinned_cores: 1,
+});
+
+export const podNumaHugepage = define<PodNumaHugepageMemory>({
+  allocated: 1024,
+  free: 2048,
+  page_size: 4068,
+});
+
+export const podNumaMemory = define<PodNumaMemory>({
+  general: {
+    allocated: 1024,
+    free: 2048,
+  },
+  hugepages: () => [],
+});
+
+export const podNuma = define<PodNuma>({
+  cores: {
```

Comment:
Same thing here, it might complain that this is hardcoded.

Reply:
Fixed here too